src/mlpack/core/cv/metrics/r2_score.hpp

```diff
@@ -43,11 +43,13 @@ namespace cv {
  * For example, a model having R2Score = 0.85, explains 85 \% variability of
  * the response data around its mean.
  */
+
+template<bool AdjustedR2>
 class R2Score
 {
  public:
   /**
-   * Run prediction and calculate the R squared error.
+   * Run prediction and calculate the R squared or Adjusted R sauared error.
```

Comment:
```suggestion
   * Run prediction and calculate the R squared or Adjusted R squared error.
```
Quick typo fix. :)

Reply:
Sorry, this was probably because I am still not very used to using an AZERTY keyboard.

Reply:
Oh wow, I've never used an AZERTY keyboard.  I think that would be really hard on my hands. :smile:

Reply:
Yes, nothing says welcome to France better than this. :)